core-model-test/framework/src/main/java/org/jboss/as/core/model/test/CoreModelTestDelegate.java

```diff
@@ -615,7 +615,7 @@ private ModelNode removeForIntellij(ModelNode model){
             //formatted strings. Simply remove it.
             //Also do the same for the following system property in the runtime platform mbean system properties:
             //"org.jboss.model.test.maven.repository.urls" => "${org.jboss.model.test.maven.repository.urls}"
-            ModelNode runtime = findModelNode(model, HOST, "master", CORE_SERVICE, PLATFORM_MBEAN, TYPE, "runtime");
```

Comment:
This is harmless, but it is not related to the scope of this Jira.

Reply:
@yersan I am not able to find a change here. This is the same as in the main branch.

Reply:
Understood. PR updated. Thanks @yersan